src/main/java/io/dockstore/provision/GSPlugin.java

```diff
@@ -110,6 +114,16 @@ protected String getBlobName(String path) {
             return String.join(File.separator, splitPathListNoBucket);
         }
 
+        private int getBufferSizeToUse() {
```

Comment:
I think this method should be commented with a reference to why we are doing this, includinga reference to the GCS library GitHub issue. That way if one year from now their ticket gets fixed, we will understand that we can remove this workaround

Reply:
comment added